### PR TITLE
Bluetooth: hci: add HCI_Write_Class_Of_Device

### PR DESCRIPTION
**Pull request by MarkWangChinese**
add the op code and the parameter structure.

Signed-off-by: Mark Wang <yichang.wang@nxp.com>

### CODE REVIEW

File: include/bluetooth/hci.h [hunk 1]
```diff
@@ -484,6 +484,11 @@ struct bt_hci_write_local_name {
 #define BT_BREDR_SCAN_INQUIRY                   0x01
 #define BT_BREDR_SCAN_PAGE                      0x02
 
+#define BT_HCI_OP_WRITE_CLASS_OF_DEVICE        BT_OP(BT_OGF_BASEBAND, 0x0024)
+struct bt_hci_cp_writ_class_of_device {
```
Comment (joerchan) [inline include/bluetooth/hci.h:488]:
```suggestion
struct bt_hci_cp_write_class_of_device {
```

**Review by joerchan — CHANGES_REQUESTED** [review]

File: include/bluetooth/hci.h (hunk 1 above)
Comment (MarkWangChinese) [reply include/bluetooth/hci.h]: updated. Thanks.

File: include/bluetooth/hci.h [hunk 2]
```diff
@@ -484,6 +484,11 @@ struct bt_hci_write_local_name {
 #define BT_BREDR_SCAN_INQUIRY                   0x01
 #define BT_BREDR_SCAN_PAGE                      0x02
 
+#define BT_HCI_OP_WRITE_CLASS_OF_DEVICE        BT_OP(BT_OGF_BASEBAND, 0x0024)
```
Comment (jhedberg) [inline include/bluetooth/hci.h:487]: minor: please indent the BT_OP... identically to the rest of this file. It seems you're missing one more space before it.

**Review by joerchan — APPROVED** [review]

File: include/bluetooth/hci.h (hunk 2 above)
Comment (MarkWangChinese) [reply include/bluetooth/hci.h]: Updated, thanks.

**Review by jhedberg — APPROVED** [review]

**Pull request merged**